server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -205,7 +205,10 @@ public enum MetadataVersion {
     IBP_3_8_IV0(20, "3.8", "IV0", true),
 
     // Introduce version 1 of the GroupVersion feature (KIP-848).
-    IBP_4_0_IVO(21, "4.0", "IV0", false);
+    IBP_4_0_IVO(21, "4.0", "IV0", false),
+
+    // Introduce version 1 of the TransactionVersion feature (KIP-890).
+    IBP_4_0_IV1(22, "4.0", "IV1", false);
```

Comment:
Do we need a separate MV version?  We could bind TV to IBP_4_0_IVO.  It's still in the future, so this should be ok as far as the bootstrap is concerned.

Reply:
I think for now the current plan is to release both in 4.0, we could reflect this in the code (if the plans change we could change the code accordingly).

Reply:
Sorry I misunderstood your question. I thought you were referring to the TVs being the same. I think we should keep separate for now since I assume these will be released at different times. But don't feel super strongly.